modules/social_features/social_tagging/src/Form/SocialTaggingSettingsForm.php

```diff
@@ -219,6 +239,17 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       $config->clear('use_category_parent')->save();
     }
 
+    foreach ($form_state->getValue('categories_order') as $tid => $categories_order_values) {
+      $term = Term::load($tid);
```

Comment:
Use dependency injection here, since we already are loading the entity type manager.

Reply:
done